pkg/apis/duck/v1alpha1/channelable_types.go

```diff
@@ -45,6 +46,12 @@ type Channelable struct {
 // ChannelableSpec contains Spec of the Channelable object
 type ChannelableSpec struct {
 	SubscribableTypeSpec `json:",inline"`
+
+	// ErrorChannel is the channel receiving messages that couldn't be sent to
+	// subscribers. One message is sent to this channel per failing subscriber.
+	//
+	// +optional
+	ErrorChannel *apisv1alpha1.Destination `json:"errorChannel,omitempty"`
```

Comment:
For each channel object, I can do a "custom" error channel? :thinking:

Reply:
Yes, for security reason. Potentially each channel is associated with an ACL and by default the error channel should have the same ACL. In practice, most channels send failed messages to a globally shared error channel but we need to allow for restricted channels.

Does that make sense to you?

Reply:
yes, that makes sense.